VideoReceiverApp/main.cpp

```diff
@@ -368,6 +347,44 @@ VideoReceiverApp::exec()
         }
      });
 
+    QObject::connect(_receiver, &VideoReceiver::onStartComplete, [this](VideoReceiver::STATUS status){
+        if (status != VideoReceiver::STATUS_OK) {
+            qCDebug(AppLog) << "Video receiver start failed";
+            _dispatch([this](){
+                if (--_connect > 0) {
```

Comment:
For the name of readability, please don't do 

`--_connect > 0`

at least put parenthesis

`(--_connect) > 0 `

or 
```

conect -= 1; 
if (_connect > 0)
```

This is not a code that needs the extra speed of picoseconds gained by --, and it's easy to miss the difference between --i and i--, plus the fact that the variable has a `_` in the front of the name --_ reminds me of the joke `what operator --> does in c++?` `if ( _connect --> 0) `